src/components/DatePicker/DatePickerHeader/DatePickerHeader.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { DropdownChevronDown, DropdownChevronUp } from "../../Icon/Icons";
+import moment from "moment";
+import { Moment } from "../types";
+import styles from "./DatePickerHeader.module.scss";
+import Icon from "../../Icon/Icon";
+
+interface DatePickerHeaderProps {
+  currentDate: Moment;
+  isMonthYearSelection: boolean;
+  onToggleMonthYearPicker: () => void;
+  hideNavigationKeys: boolean;
+  "data-testid"?: string;
+}
+
+const DatePickerHeader = (props: DatePickerHeaderProps) => {
+  const {
+    currentDate,
+    isMonthYearSelection,
+    onToggleMonthYearPicker,
+    hideNavigationKeys,
+    "data-testid": dateTestId
+  } = props;
+
+  const localedDated = moment(currentDate.valueOf());
+  const month = localedDated.format("MMMM");
+  const year = localedDated.format("YYYY");
+  const string = month + " " + year;
+  return (
+    <div className={styles.datePickerHeaderContainer}>
+      <div>{string}</div>
+      {!hideNavigationKeys && (
+        <button
```

Comment:
For next time if you want to implement a button which not looks like a button, we have a "clickable" component in vibe for this case :) 
I think it's ok to keep it like this for now (you don't need to refactor this right now), it's not a blocker because this button is accessible because it's a native button.